packages/ariakit-react-core/src/select/select.tsx

```diff
@@ -202,6 +202,7 @@ export const useSelect = createHook<TagName, SelectOptions>(function useSelect({
             name={name}
             form={form}
             required={required}
+            disabled={props.disabled}
```

Comment:
I believe this should be added to the dependencies array.

Reply:
Done!